bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/references/js/wicket.tempus-dominus.js

```diff
@@ -0,0 +1,19 @@
+function createTempusDominus(elementId, config, localization, lang) {
```

Comment:
IMO it would be good to add these functions to `Wicket.Bootstrap` object. Currently they pollute the global space

Reply:
fixed!